### DISCUSSION TITLE
loading/error.tsx ignored for parallel routes

### DISCUSSION THREAD

**Discussion by josias-r** [vercel/next.js#50107] [Help] [answered]
### Summary

From your docs:

> Parallel Routing allows you to define independent error and loading states for each route as they're being streamed in independently.

I have a parallel route setup (`/app/@feature`), and the loading state is not shown. During development or for a built project.

The loading.tsx file from the parent page is shown instead if it exists, but never the loading.tsx of the parallel route.

Sounds to me like the docs promise more than there really is :)

Are there limitations I am not aware of or is that simply not implemented?

**Edit:**

Same goes for `error.tsx` files :)

**Comment by aspirisen** [comment, marked as answer]
Probably related issue https://github.com/vercel/next.js/issues/49243

**Comment by T04435** [comment]
FYI: this was an issue in 13.4.4 updated to @latest(13.4.16) and it all works as per [parallel route docs](https://nextjs.org/docs/app/building-your-application/routing/parallel-routes)

**Comment by yabuking84** [comment]
Its happening in 14.1.2

**Comment by sim391calado** [comment]
And still on 14.3.0

**Comment by Abukodonosor** [comment]
Problem in next15@rc!
